lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
This is a special case.  I'd like to have @andy-armstrong weigh in on which direction he would go.  As I said, I would lean toward #2 in my earlier comment: https://github.com/edx/edx-platform/pull/11214#discussion-diff-49631289

Let's see if he leans towards #1 or #2.  Thanks.